networks/suzuka/suzuka-full-node/src/main.rs

```diff
@@ -14,13 +14,21 @@ async fn main() -> Result<(), anyhow::Error> {
 			.init();
 	}
 
+	// Load variables defined in .env file.
+	let movement_storage_path =
+		std::env::var("MOVEMENT_BASE_STORAGE_PATH").unwrap_or("".to_string());
+	let mut env_file_path = std::env::current_dir()?;
+	env_file_path.push(movement_storage_path);
+	env_file_path.push(".env".to_string());
+	dotenv::from_filename(env_file_path)?;
```

Comment:
I think we should not depend on `dotenv` in production code. Setting environment variables in the process is [generally unsafe](https://doc.rust-lang.org/std/env/fn.set_var.html#safety), and even if it's safe in this particular case, it's a deeply weird way to configure the node.

Can we just source the .env file in the suzuka-full-node shell script? Not that it should be even necessary if you can just get the prelude script to set the environment variables for process-compose.